changelog.dd

```diff
@@ -6,28 +6,13 @@ $(COMMENT Pending changelog for 2.071. This will get copied to dlang.org and
 
 $(BUGSTITLE Library Changes,
 
-$(LI $(RELATIVE_LINK2 json-encode-control-characters, `std.json` encodes string control characters.))
-$(LI The package $(STDMODREF experimental_ndslice, std.experimental.ndslice) was added.
-It is also available as part of the $(LINK2 https://github.com/DlangScience/mir, Mir library).)
-$(LI Default LogLevel of FileLogger was changed to LogLevel.all.)
-$(LI Access to the internal `ptr` and `len` fields of $(XREF bitmanip, BitArray) was deprecated.)
-$(LI $(XREF experimental.allocator.gc_allocator, GCAllocator.goodAllocSize)
+$(LI $(REF GCAllocator.goodAllocSize, std,experimental,allocator,gc_allocator)
```

Comment:
You sure this is how it is supposed to be? This file doesn't get built by @CyberShadow's autotester, so it's hard for me to know whether this works.

I find it hard to believe that this needs 5 parameters.